### DISCUSSION TITLE
How to implement undo/redo in react-data-grid?

### DISCUSSION THREAD

**Discussion by aubeena007** [Comcast/react-data-grid#2437] [Q&A]
i want to implement undo/redo in react data grid . i could not find any props or call back in official website undo : control+z redo : control+y

**Comment by devChedar** [comment]
@aubeena007 by any chance did you implement this on your own?

**Reply by dgdrake** [reply]
Curious to know if there was any updates on this topic? Was there an implementation for this?

**Reply by devChedar** [reply]
React data grid doesn't support it. I managed to get the feature working using redux undo.  You can take a look into that.
